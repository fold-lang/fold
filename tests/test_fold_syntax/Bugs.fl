// Nested quoting
let x = `{ $ (f `x ) };

// Handling of arbitrary infix ops in juxt.
let x = f a |> g

