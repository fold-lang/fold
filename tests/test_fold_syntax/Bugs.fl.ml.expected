let x = Scope ("{", (f (Ident (Lower "x"))), "}")
let x = (f a) |> g
